### PR TITLE
Testing improve garage package

### PR DESCRIPTION
**Pull request by oiseauroch**

### CODE REVIEW

**Comment by oiseauroch** [comment]
!testme

**Comment by oiseauroch** [comment]
!testme

**Comment by oiseauroch** [comment]
try to fix #4 . @alexAubin how should I remove the custom nginx conf when removing the app ?

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -292,6 +284,19 @@ then
 	VIRTUALISATION=true $final_path/umount_disk.sh
 fi
 
+#=================================================
+# ADD REGEN-CONF HOOK
+#=================================================
+ynh_script_progression --message="adding regen-conf hook..."  --weight=1
+
+ ynh_add_config --template="../sources/hooks/conf_regen/98-nginx_garage" --destination="/usr/share/yunohost/hooks/conf_regen/98-nginx_$app"
```
Comment (alexAubin) [inline scripts/install:292]: It sounds like you're missing a `yunohost tools regen-conf nginx` here ?

**Comment by alexAubin** [comment]
@oiseauroch 

>try to fix https://github.com/YunoHost-Apps/garage_ynh/issues/4 . @alexAubin how should I remove the custom nginx conf when removing the app ?

(cf also https://github.com/YunoHost-Apps/garage_ynh/issues/4#issuecomment-1454070285 )

I guess you found out since then that the hook should be named `XX-nginx_$app`, which is a very poorly documented (or probably undocumented..) bit of logic such that `yunohost tools regen-conf nginx` will call both the base hook and the custom hook

Apart from the maybe missing call to `yunohost tools regen-conf nginx` in the install script, that PR looks okay to me :+1:

**Comment by oiseauroch** [comment]
!testme

**Pull request merged**